mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -185,6 +186,51 @@ void AbstractControllerExecution::setVelocityCmd(const geometry_msgs::TwistStamp
   // TODO so there should be no loss of information in the feedback stream
 }
 
+bool AbstractControllerExecution::checkVelocityIgnore(const bool& robot_stopped,
+                                                      const geometry_msgs::TwistStamped& cmd_velocity)
+{
+  bool ignored_exceed_tolerance = false;
+
+  if (robot_stopped)
+  {
+    // linear total and angular
+    double cmd_linear;
+    double cmd_angular;
+
+    // compute linear velocity magnitude
+    cmd_linear = hypot(cmd_velocity.twist.linear.x, cmd_velocity.twist.linear.y);
+    // compute angular velocity magnitude
+    cmd_angular = fabs(cmd_velocity.twist.angular.z);
```

Comment:
```suggestion
    // compute linear velocity magnitude
    const double cmd_linear = std::hypot(cmd_velocity.twist.linear.x, cmd_velocity.twist.linear.y);
    
    // compute angular velocity magnitude
    const double cmd_angular = std::abs(cmd_velocity.twist.angular.z);
```
S: declare as const and init in same line
F: use `std::hypot` and `std::abs` instead